cpu/samd5x/Makefile.dep

```diff
@@ -1 +1,6 @@
+# use tamper detection to wake from Deep Sleep
+ifneq (,$(filter periph_gpio_irq,$(USEMODULE)))
+  USEMODULE += periph_rtc_rtt
+endif
```

Comment:
This dependency does not seem correct IMO.

Reply:
How else could it look like?

Reply:
Is what I mentioned in the main comment, I would have a distinct pseudomodule for this.

Reply:
Ok, changed it.